TestingAttributes/TestingAttributes/Program.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TestingAttribute.AttributeRetreival;
+
+namespace TestingAttributes
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+                string category;
```

Comment:
This implementation is wrong.
The Dll path and the category will come as command line arguments.
Check if class has TestClass attribute
If present Check every method for TestMethod attribute else ignore the class.
If present check if ignore Attribute, if present add this to the ignored test list
else add it to the methods to be executed list.